YIF.Core.Domain/YIF.Core.Domain.csproj

```diff
@@ -16,7 +16,7 @@
   <ItemGroup>
     <ProjectReference Include="..\YIF.Core.Data\YIF.Core.Data.csproj" />
     <PackageReference Include="AutoMapper" Version="10.1.1" />
-    <PackageReference Include="FluentValidation.AspNetCore" Version="9.5.3" />
+    <PackageReference Include="FluentValidation.AspNetCore" Version="10.0.4" />
```

Comment:
Is it ok, that you've updated fluentValidation?

Reply:
I`ve just had issues wich are resolved only after reinstalling the package, so i think it`s ok